packages/experimental/Avatar/FluentUIReactNativeAvatar.podspec

```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do |s|
 
   s.ios.deployment_target = "13.0"
   s.ios.source_files      = "ios/*.{swift,h,m}"
-  s.ios.dependency 'MicrosoftFluentUI', '~> 0.2.2'
+  s.ios.dependency 'MicrosoftFluentUI', '~> 0.2.6'
```

Comment:
why is this 0.2.6 while others are 0.2.7 in pod.lock file?

Reply:
`~> 0.2.6` resolves to the latest version of `0.2.x`, which at the time of running `pod install` was 0.2.7